### PR TITLE
Universal/docr 479 new UI changes

### PR DESCRIPTION
**Pull request by jpriyas**
This PR adds changes specific to the URP Asset page to reflect the new UI introduced in 2021.2.

### CODE REVIEW

File: com.unity.render-pipelines.universal/Documentation~/universalrp-asset.md [hunk 1]
```diff
@@ -10,12 +10,11 @@ You can have multiple URP assets and switch between them. For example, you can h
 ## UI overview
 In the URP, you can configure settings for:
 
-- [__Rendering__](#rendering)
+- [__Rendering__](#renderingl)
```
Comment (oleks-k) [inline com.unity.render-pipelines.universal/Documentation~/universalrp-asset.md:13]: Is this change made on purpose?

File: com.unity.render-pipelines.universal/Documentation~/universalrp-asset.md [hunk 2]
```diff
@@ -76,6 +78,8 @@ If you disable some of these settings, the relevant [keywords](shader-stripping.
 | __Main Light__        | These settings affect the main [Directional Light](https://docs.unity3d.com/Manual/Lighting.html) in your Scene. You can select this by assigning it as a [Sun Source](https://docs.unity3d.com/Manual/GlobalIllumination.html) in the Lighting Inspector. If you don’t assign a sun source, the URP treats the brightest directional light in the Scene as the main light. You can choose between [Pixel Lighting](https://docs.unity3d.com/Manual/LightPerformance.html) and _None_. If you choose None, URP doesn’t render a main light,  even if you’ve set a sun source. |
 | __Cast Shadows__      | Check this box to make the main light cast shadows in your Scene. |
 | __Shadow Resolution__ | This controls how large the shadow map texture for the main light is. High resolutions give sharper, more detailed shadows. If memory or rendering time is an issue, try a lower resolution. |
+| __Mixed Lighting__    | Enable [Mixed Lighting](https://docs.unity3d.com/Manual/LightMode-Mixed.html), to tell the pipeline to include mixed lighting shader variants in the build. |
```
Comment (oleks-k) [inline com.unity.render-pipelines.universal/Documentation~/universalrp-asset.md:81]: Suggestion:
With [Mixed Lighting](https://docs.unity3d.com/Manual/LightMode-Mixed.html) enabled, Unity includes mixed lighting shader variants in the build.

File: com.unity.render-pipelines.universal/Documentation~/universalrp-asset.md (hunk 1 above)
Comment (jpriyas) [reply com.unity.render-pipelines.universal/Documentation~/universalrp-asset.md]: No, it was a typo. I've fixed in the new version.

**Review by oleks-k — APPROVED** [review]
Looks good!

**Review by manuele-bonanno — APPROVED** [review]

**Pull request merged**